api/File.json

```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://w3c.github.io/FileAPI/#file-section",
         "support": {
           "chrome": {
-            "version_added": "13"
+            "version_added": "1"
```

Comment:
Shouldn't some of the members also be supported in Chrome 1?

Reply:
It looks like pretty much all the members are actually not defined in Chrome 13, even.  In these PRs, though, I'm only focusing on changes where the version number reports earlier than what BCD says, and leaving the other direction for another time -- I'll be sure to look into the members in the next wave!

Reply:
If none of the members were present in Chrome 1, then there's a good chance that the `File` interface didn't actually work and should be `partial_implementation` and/or have some notes. I don't think we should update part of the data and leave it less consistent based just on a `"File" in self` test without understanding if the inconsistency is correct or not.